portfolio/src/main/webapp/script.js

```diff
@@ -66,7 +66,12 @@ function tryHarderAndGiveDaisy() {
 
 async function getHelloFromButton() {
     const response = await fetch('/data');
-    const text = await response.text();
-    const cutieTextContainer = document.getElementById('cutie-text');
-    cutieTextContainer.innerText = text;
+    const text = await response.json();
+    for (i = 0; i < text.length; i++){
+        var messageToDisplay = text[i];
```

Comment:
Use `const` for variables that are never reassigned. Use `let` otherwise. Never use `var`.